.styleci.yml

```diff
@@ -0,0 +1,6 @@
+preset: psr2
+
+enabled:
+  - short_array_syntax
```

Comment:
Good catch @aidantwoods , changed that in https://github.com/aidantwoods/SecureHeaders/pull/9/commits/159578149bc8b884898642ec9837292983adb035